posix.mak

```diff
@@ -525,6 +525,17 @@ style: ../dscanner/dsc $(LIB)
 	@echo "Enforce space between binary operators"
 	grep -nrE "[[:alnum:]](==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]] (==|!=|<=|<<|>>|>>>|^^)[[:alnum:]]|[[:alnum:]](==|!=|<=|<<|>>|>>>|^^) [[:alnum:]]" $$(find . -name '*.d'); test $$? -eq 1
 
+	@echo "Validate changelog files (Do _not_ use REF in the title!)"
```

Comment:
This is consistent with all other checks. If you want to make the checks less noisy, I am happy to do so, but then we should do this for all checks and in another PR.